app/assets/stylesheets/views/mainstream_browse_pages.scss

```diff
@@ -1,12 +1,17 @@
-/* import govuk_admin_template to get bootstrap styles */
+// import govuk_admin_template to get bootstrap styles
 @import 'govuk_admin_template';
 
-table.tags-list {
+$light-grey: #eee;
+$grey: #888;
```

Comment:
Is it confusing that in one file we declare `$grey` as one colour, but declare it as another colour in another file?

Reply:
I'm happy for this to go as is to set-up linting, but create another task to consolidation the colour scheme.